### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by curran** [unhcr/dataviz-population-change#68]
There are a few minor issues blocking a proper release of this:

 * [ ] Bug with increase/decrease text on year slider event #28

**Comment by Edouard-Legoupil** [comment]
shouldn't you add: https://github.com/unhcr/dataviz-population-change/issues/67?

**Comment by matthewsmawfield** [comment]
I agree with the suggestion, however, the current scope is  on rebranding the existing visualisation - here: http://data.unhcr.org/dataviz/
